spec/models/fax_recipient_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe FaxRecipient do
+  subject(:instance) do
```

Comment:
generally prefer to not use `subject` or `let` in RSpec tests if that's OK with you. We have it in some places so they can be used in shared examples but I find these aspects of the RSpec DSL to make tests harder to read and update.